pulp_ansible/app/global_access_conditions.py

```diff
@@ -1,15 +1,17 @@
+from pulp_ansible.app import models
+
+
 def v3_can_view_repo_content(request, view, action):
```

Comment:
So here I would either modify this method to accept another argument for the **repository** permission you want to check against. Or just create another method with similar logic for extracting the repository and testing the permission against it and that should be the access condition used in your V3 access policies.